mmv1/third_party/terraform/resources/resource_compute_network_peering.go.erb

```diff
@@ -104,6 +105,14 @@ func ResourceComputeNetworkPeering() *schema.Resource {
 				Computed:    true,
 				Description: `Details about the current state of the peering.`,
 			},
+
+			"stack_type": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: validateEnum([]string{"IPV4_ONLY", "IPV4_IPV6", ""}),
```

Comment:
Just a question on this: why is the `""` needed as an option? It seems like the flatten function ensures that we never keep `""` as a value in the terraform state.

Reply:
Fixed it. Thanks!